packages/@okta/vuepress-site/docs/reference/api/idps/index.md

```diff
@@ -455,6 +457,158 @@ For example: `?RelayState=:anyUrlEncodedValue`
 The deep link for the above three parts is:<br>
 `https://{myOktaDomain}.com/sso/saml2/:idpId/app/:app-location/:appId/sso/saml?RelayState=:anyUrlEncodedValue`
 
+#### Add Apple Identity Provider
+
+Adds a new `Apple` type IdP to your organization
+
+##### Request example
+
+```bash
+curl -v -X POST \
+-H "Accept: application/json" \
+-H "Content-Type: application/json" \
+-H "Authorization: SSWS ${api_token}" \
+-d '{
+  "type": "APPLE",
+  "name": "Apple",
```

Comment:
nit: flesh out the name a bit to distinguish it from `type`, for instance "Apple Identity Provider"